src/node/cli.ts

```diff
@@ -37,6 +40,7 @@ Options:
   --sourcemap                [boolean] output source maps for build (default: false)
   --minify                   [boolean | 'terser' | 'esbuild'] enable/disable minification, or specify
                                        minifier to use. (default: 'terser')
+  --mode, -m                 [string]  specify env mode (default: ${defaultMode})
```

Comment:
Should directly show default as `'development' for dev, 'production' for build` since the help message may be printed without a command.

Reply:
👌